doc/source/release-notes/Percona-Server-5.6.24-72.2.rst

```diff
@@ -0,0 +1,42 @@
+.. rn:: 5.6.24-72.2
+
+==============================
+ |Percona Server| 5.6.24-72.2 
+==============================
+
+Percona is glad to announce the release of |Percona Server| 5.6.24-72.2 on May 8th, 2015 (Downloads are available `here <http://www.percona.com/downloads/Percona-Server-5.6/Percona-Server-5.6.24-72.2/>`_ and from the :doc:`Percona Software Repositories </installation>`).
+
+Based on `MySQL 5.6.24 <http://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-24.html>`_, including all the bug fixes in it, |Percona Server| 5.6.24-72.2 is the current GA release in the |Percona Server| 5.6 series. All of |Percona|'s software is open-source and free, all the details of the release can be found in the `5.6.24-72.2 milestone at Launchpad <https://launchpad.net/percona-server/+milestone/5.6.24-72.2>`_.
+
+New Features
+============
+
+ TokuDB storage engine package has been updated to version `7.5.7 <http://docs.tokutek.com/tokudb/tokudb-release-notes.html#tokudb-version-7-x>`_.
+ 
+Bugs Fixed
+==========
+
+ A server binary as distributed in binary tarballs could fail to load on different systems due to an unsatisfied ``libssl.so.6`` dynamic library dependency. This was fixed by replacing the single binary tarball with multiple tarballs depending on the *OpenSSL* library available in the distribution: 1) ssl100 - for all *Debian/Ubuntu* versions except *Squeeze/Lucid* (``libssl.so.1.0.0 => /usr/lib/x86_64-linux-gnu/libssl.so.1.0.0 (0x00007f2e389a5000)``); 2) ssl098 - only for *Debian Squeeze* and *Ubuntu Lucid* (``libssl.so.0.9.8 => /usr/lib/libssl.so.0.9.8 (0x00007f9b30db6000)``); 3) ssl101 - for *CentOS 6* and *CentOS 7* (``libssl.so.10 => /usr/lib64/libssl.so.10 (0x00007facbe8c4000)``); 4) ssl098e - to be used only for *CentOS 5* (``libssl.so.6 => /lib64/libssl.so.6 (0x00002aed5b64d000)``). Bug fixed :bug:`1172916`.
+ 
+ Execution of stored procedures with subqueries leads to memory leaks. Bug fixed :bug:`1380985` (upstream :mysqlbug:`76349`).
+
+ Multi-threaded replication on slave would break with an ``1755`` error after restarting the server a few times. This is a regression introduced by fix for bug :bug:`1331586` in :rn:`5.6.21-70.0`.
```

Comment:
"A slave server restart could cause a 1755 slave SQL thread error if multi-threaded slave was enabled. This was a regression ... "

also missing bug ref